packages/zowe-explorer/i18n/sample/src/dataset/actions.i18n.json

```diff
@@ -91,6 +91,7 @@
   "paste.dataSet": "Invalid paste. Copy dataset(s) first",
   "paste.dataSet.InPrg": "Copying File(s)",
   "download.invalidNode": "downloadDataset() called with invalid node. ",
+  "copyDataSet.error.notSupported": "This operation is not supported at this time.",
```

Comment:
Do we want to make this more specific to the "copy dataset" action? Something like "Copying datasets is not supported" - not a request to change, just wanted to get your thoughts on it